payment-handler/throw-error-if-redundant-sw-manual.https.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<meta charset=utf-8>
+<link rel="help" href="https://w3c.github.io/payment-handler/">
+<title>Throw InvalidStateError if redundant service worker</title>
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<script src="/service-workers/service-worker/resources/test-helpers.sub.js"></script>
+<script>
+
+promise_test(async test => {
+  const permission = PaymentManager.requestPermission();
```

Comment:
Chrome does not yet have permission request implemented and this test is not about permission requests. Can we feature-check for it here? 

```javascript
if (PaymentManager.requestPermission) {
  const permission = PaymentManager.requestPermission();
  assert_equals(permission, 'granted');
}

Reply:
IMHO, we should write the tests in the right way even if it's not currently implemented yet.
If the permission is not granted, instruments.set() SHOULD be failed according to the spec.
As you know, if this test sync-up with Chromium, will generate failed -expected.txt automatically. So, it's not problem.

(BTW, I left a comment in https://chromium-review.googlesource.com/c/chromium/src/+/533193)

Reply:
Sounds good.